packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -34,7 +35,13 @@ export type ResponseMessage = {
   tips: Map<string, CID>
 }
 
-export type PubsubMessage = UpdateMessage | QueryMessage | ResponseMessage
+// All nodes will always ignore this message
+export type KeepaliveMessage = {
+  typ: MsgType.KEEPALIVE
+  ts: number // current time
```

Comment:
Why do we include a timestamp?

Reply:
Not strictly needed.  Just keeps each keepalive message unique, and figured it might be useful in debugging if we have connectivity issues in the future, it would be easier to see when we last published a message

Reply:
Ok, just thinking this adds to bandwidth overhead, but maybe just marginal?